galaxy_ng/app/settings.py

```diff
@@ -30,6 +30,10 @@
 # created by the initial-data.json data fixture
 GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH = "automation-hub"
 
+# The format for the name of the per account synclist, and the
+# associated repository, distribution, and distribution base_paths
+GALAXY_API_SYNCLIST_NAME_FORMAT = "{account_name}-synclist"
```

Comment:
Since this will also be the repo name... should we align the "type" and the "identifier" format for different repo types? ie either
1. `inbound-{namespace}` and `synclist-{account_name}` ... or ...
2. `{namespace}-inbound` and `{account_name}-synclist`

I'm kinda partial to the first one